pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientNameEntity.java

```diff
@@ -0,0 +1,224 @@
+package gov.ca.emsa.pulse.broker.entity;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+@Entity
+@Table(name="patient_name")
+public class PatientNameEntity {
+	
+	@Id 
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column( name = "id", nullable = false )
+	private Long id;
+	
+	@Column(name = "given_name_id")
+	private Long givenNameId;
+	
```

Comment:
I don't think the `given_name_id` should be here. The GivenNameEntity description should have a `patient_id` foreign key relationship, but we don't need one going the other way